api/src/test/java/org/openmrs/module/xstream/ConceptClassSerializationTest.java

```diff
@@ -108,7 +108,6 @@ public void shouldDeserializeConceptClass() throws Exception {
 		xmlBuilder.append("    </person>\n");
 		xmlBuilder.append("    <systemId>1-8</systemId>\n");
 		xmlBuilder.append("    <username>admin</username>\n");
-		xmlBuilder.append("    <secretQuestion></secretQuestion>\n");
```

Comment:
Did you intentionally remove the above line?

Reply:
Yeah. Keeping this gives an error.

`com.thoughtworks.xstream.mapper.CannotResolveClassException: secretQuestion`

logs: https://pastebin.com/q860JzW3

Reply:
Because it was removed from openmrs-core 2.0.0 is what i expected 😊

Reply:
Oh I thought that was obivious :)